src/Root.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  HashRouter, Navigate, Route, Routes,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './components/HomePage';
+import { PeopleList } from './components/PeopleList';
+import { PageNotFound } from './components/PageNotFound/PageNotFound';
+
+export const Root = () => (
+  <HashRouter>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="people" element={<PeopleList />}>
+          <Route path="/people/:slug?" element={<PeopleList />} />
```

Comment:
You don't need to repeat 'people' here, no need to insert element in the upper line, just add .. to the link in PersonLink like this: (to={`../${slug}`})